.docker/parameters.yml

```diff
@@ -34,6 +34,7 @@ parameters:
     rds_articles:
         '45562':
             display: https://repro.elifesciences.org/example.html
```

Comment:
what about using the real links from https://hub.stenci.la/elife/30274 to provide feedback in development?

Reply:
If allowed for local development, then can do. Could always use Lens too (not RDS, but that doesn't matter).

Reply:
Lens wouldn't reproduce the iframe options security issue.